cernopendata/modules/fixtures/data/records/lhcb-antimatter-matters-2017.json

```diff
@@ -0,0 +1,198 @@
+[
+{
+  "abstract": {
+    "description": " <p>This record contains the data set that is made available for the analysis described in the Project Notebook. The data was collected using the LHCb experiment in 2011 at a centre-of-mass energy of 7 TeV. The data has been reconstructed using the software framework of the experiment. The data has been selected by the trigger of the experiment and an offline selection made. The sample provided contains the candidate events observed for the decay of a charged B meson into three charged particles. In addition to providing  the data of the experiment a file containing a simplified simulation of the decay of a charged B meson to three charged kaons is also provided. This simplified simulation file has not been passed through a simulation of the detector response.<p>\n       <p>The data files contain only the most important reconstructed variables for performing the analysis of this data set. A full description of the variables is provided in the Project Notebook. The data files are stored in the format a ROOT TTree.</p> ",
+    "links": [
+      {
+        "description": "The LHCb experiment is described in",
+        "url": "http://cds.cern.ch/record/1129809"
+      }
+    ]
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "LHCb collaboration"
+  },
+  "collections": [
+    "LHCb-Derived-Datasets"
+  ],
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "root"
+    ], 
+    "number_events": 8550000,
+    "number_files": 3,
+    "size": 1113480280
+  },
+  "doi": "10.7483/OPENDATA.LHCB.AOF7.JH09",
+  "experiment": "LHCb",
+    "files": [
+    {
+      "size": 666484974, 
+      "type": "xrootd", 
```

Comment:
Please add `checksum` property to all `file` field instances:

```
 "checksum": "sha1:0000000000000000000000000000000000000000", 
```

Reply:
Done